### DISCUSSION TITLE
How to upgrade to lastest frigate version (14.0, with the new UI)

### DISCUSSION THREAD

**Discussion by MaximCrabbe** [tteck/Proxmox#3511] [Help Needed]
I installed frigate LXC (13.2) a few months ago with its helper script with default settings. I can ofc apt-get update/upgrade etc, but frigate stays on the same version. (Yes I understand now it is not managed on the apt-get repo).
When I google to upgrade frigate, all examples are with docker compose/docker. Docker is not installed in the frigate LXC so this is not the way to update it. Any guidance would be appreciated.

**Comment by tteck** [comment]
Create a new Frigate LXC and transfer configuration.
